core/modules/node/tests/node.test

```diff
@@ -525,7 +538,7 @@ class NodeCreationTestCase extends BackdropWebTestCase {
     $this->backdropPost('node/add/page', $edit, t('Save'));
 
     // Check that the user was redirected to the home page.
-    $this->assertText(t('Welcome to Backdrop'), 'The user is redirected to the home page.');
+    $this->assertText(t('This is your first post, edit it or delete it'), 'The user is redirected to the home page.');
```

Comment:
Oh boy. This seems less than ideal check for the homepage. In any case, missing a period at the end of this sentence, which will cause a new translated string.